CHANGELOG.rst

```diff
@@ -2,6 +2,22 @@
 SDK Changelog
 =============
 
+2.3.0-alpha.0 (2024-05-01)
+==========================
+
+First preview release of the upcoming 2.3.0 version.
+
+* [#4009] Fix summary display of `fieldset`
+* [#4082] Fix duplicate creation of submissions
+* [#4172] Fix parsing of reference dates
+* [#4130] Do not show main url when asking for payment
+* [#4115] Updates to GovMetric to account for abort and feedback IDs
+* [#4208] Embed Govmetric imgs
+* [#4201] Fix crash with hidden map component
+* [#4222] Fix max files upload limit
+* [#4220] Fix "optional" translation for grouped components
+* [#4207] Fix styling overflow for select dropdown
```

Comment:
Please apply consistent language style:

* "Fixed" instead of "fix"
* "Added" instead of "add"
* Entries end with a <kb>.</kb>, treat them like sentences